From 002afdd5de52f4819b376dcd24cdb1755509e6cf Mon Sep 17 00:00:00 2001
From: Corinna Vinschen <corinna@vinschen.de>
Date: Thu, 20 May 2021 10:03:34 +0200
Subject: [PATCH 38/N] Cygwin: Add missing IsWow64Process2 declaration

The previous patch is accidentally missing the declaration of
IsWow64Process2.  Add it belatedly.

Fixes: 1865a41cb383 ("Cygwin: suppress FAST_CWD warnings on ARM64")
Signed-off-by: Corinna Vinschen <corinna@vinschen.de>
---
 winsup/cygwin/winlean.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/winsup/cygwin/winlean.h b/winsup/cygwin/winlean.h
index 2e2308e..3382686 100644
--- a/winsup/cygwin/winlean.h
+++ b/winsup/cygwin/winlean.h
@@ -110,6 +110,9 @@ extern "C" {
    executed on systems that support it (a run-time condition). */
 PVOID WINAPI VirtualAlloc2 (HANDLE, PVOID, SIZE_T, ULONG, ULONG,
 			    PMEM_EXTENDED_PARAMETER, ULONG);
+/* IsWow64Process2 should be declared in <w32api/wow64apiset.h> but
+   isn't yet. */
+BOOL WINAPI IsWow64Process2(HANDLE, USHORT *, USHORT *);
 
 #ifdef __cplusplus
 }
-- 
2.31.1.windows.1

